app/src/main/java/de/westnordost/streetcomplete/quests/backup_generator/AddBackupGenerator.kt

```diff
@@ -0,0 +1,58 @@
+package de.westnordost.streetcomplete.quests.backup_generator
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.CITIZEN
+import de.westnordost.streetcomplete.osm.IS_SHOP_OR_DISUSED_SHOP_EXPRESSION
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.quests.YesNoQuestForm
+import de.westnordost.streetcomplete.util.ktx.toYesNo
+
+class AddBackupGenerator : OsmFilterQuestType<Boolean>() {
+    val amenities = listOf(
+        "bar", "cafe", "fast_food", "food_court", "pub", "restaurant", "cinema",
+        "nightclub", "planetarium", "theatre", "internet_cafe", "car_wash", "telephone",
+        "vending_machine"
+    )
+    val tourisms = listOf(
+        "zoo", "aquarium", "theme_park", "hotel", "hostel", "motel", "guest_house",
```

Comment:
what should be answered for say zoo that has generators to keep vulnerable animals alive but would be closed due power outage?

Reply:
You are right, it is the same with the aquarium